docs/running-notebooks/closing-active-sessions/index.md

```diff
@@ -30,3 +30,21 @@ click on the vertical ellipsis menu on the right-hand-side and select
 "Close Jupyter Notebook".
 
 ![ui-close-session](../../images/ui-close-session.png)
+
+### Auto-closure of sessions
+
+REANA provides an auto-closure feature for interactive sessions.
+Administrators can configure a maximum number of inactivity hours after
+which interactive sessions will be automatically closed.
```

Comment:
What do you think about this?

```suggestion
Administrators can configure REANA to automatically close interactive sessions after a given period of inactivity in order to free unused resources.
```

In this way, we can also remove the text at the end _This feature ensures ... utilization_